src/MLOps.NET.AWS/Storage/S3BucketModelRepository.cs

```diff
@@ -1,69 +1,114 @@
-﻿using Amazon;
-using Amazon.S3;
+﻿using Amazon.S3;
 using Amazon.S3.Model;
-using Amazon.S3.Util;
+using MLOps.NET.Entities.Impl;
+using MLOps.NET.Storage.Deployments;
 using System;
 using System.IO;
-using System.Security.AccessControl;
-using System.Threading;
-using System.Threading.Tasks;
 using System.Linq;
-using Amazon.S3.Model.Internal.MarshallTransformations;
+using System.Threading.Tasks;
 
 namespace MLOps.NET.Storage
 {
     internal sealed class S3BucketModelRepository : IModelRepository
     {
-        private readonly IAmazonS3 amazonS3Client;
-        private string bucketName;
+        private readonly IAmazonS3 s3Client;
+        private readonly IModelPathGenerator modelPathGenerator;
+        private readonly string modelRepositoryBucket = "model-repository";
+        private readonly string deploymentRepositoryBucket = "deployment";
 
-        public S3BucketModelRepository(IAmazonS3 amazonS3Client, string bucketName)
-        {           
-            this.bucketName = bucketName;
-            this.amazonS3Client = amazonS3Client;
+        public S3BucketModelRepository(IAmazonS3 amazonS3Client, IModelPathGenerator modelPathGenerator)
```

Comment:
`amazonS3Client` We changed this in to `s3Client` for consistency should we not change this too? Same in the rest of this file.

Reply:
Great catch, I'll push a change for that